File: Exactly Electrical/exactlyelectrical.js
let nums = readline().split(" ");
let x1 = parseInt(nums[0]);
let y1 = parseInt(nums[1]);

let nums2 = readline().split(" ");
let x2 = parseInt(nums2[0]);
let y2 = parseInt(nums2[1]);

let t = parseInt(readline());
let check = Math.abs(x2-x1)+Math.abs(y2-y1)-t;
console.log((check > 0 || check % 2 == 1 || check % 2 == -1) ? "N" : "Y");

File: Exactly Electrical/ExactlyElectrical.java
import java.io.*;

public class ExactlyElectrical {
    public static void main(String[] args) throws IOException {
        InputStreamReader inp = new InputStreamReader(System.in);
        BufferedReader sc = new BufferedReader(inp);
        PrintWriter writer = new PrintWriter(System.out);
        String[] nums = sc.readLine().split(" ");
        int x1 = Integer.parseInt(nums[0]);
        int y1 = Integer.parseInt(nums[1]);
        String[] nums2 = sc.readLine().split(" ");
        int x2 = Integer.parseInt(nums2[0]);
        int y2 = Integer.parseInt(nums2[1]);
        int t = Integer.parseInt(sc.readLine());
        int check = Math.abs(x2-x1)+Math.abs(y2-y1)-t;
        writer.println((check > 0 || check % 2 == 1 || check % 2 == -1) ? "N" : "Y");
        writer.flush();
    }
}